tests/wpt/web-platform-tests/html/semantics/embedded-content/the-img-element/img.complete.html

```diff
@@ -33,16 +33,20 @@
   var t = async_test("async src complete test");
 
   t.step(function(){
-    document.getElementById("imgTestTag3").src = '3.jpg?nocache=' + Math.random();
+    document.getElementById("imgTestTag3").onload = t.step_func(function(){
+      assert_true(document.getElementById("imgTestTag3").complete);
+      assert_equals(document.getElementById("imgTestTag3").currentSrc.split("/").pop().split('?')[0], "3.jpg");
```

Comment:
Actually, why don't we verify that we only see the second src value that was set?